src/App.svelte

```diff
@@ -592,6 +598,11 @@
 
     function setSelection(event_or_index) {
         let index = event_or_index.detail?.index ?? event_or_index
+        if (event_or_index.detail?.toBottom) {
```

Comment:
There's a bug where the previous regular selection stays selected if you select a new one with this method:
```suggestion
        if (event_or_index.detail?.toBottom) {
                resetSelection()
```